src/seedu/addressbook/common/UtilsTest.java

```diff
@@ -0,0 +1,57 @@
+package seedu.addressbook.common;
+
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+
+import seedu.addressbook.data.person.*;
+
+import static org.junit.Assert.*;
+
+import java.util.Arrays;
+import java.util.Collection;
+import java.util.List;
+
+public class UtilsTest {
+	
+	@Rule
```

Comment:
Indentation problem here. Our coding standard requires you to use 4 spaces instead of tabs. You can configure Eclipse to convert tabs to spaces automatically.